io/transfer/transfer.go

```diff
@@ -57,6 +58,15 @@ type TargetFilter struct {
 	Type string
 }
 
+// URLFilter filters for any [URLEvent] which the scheme matches against
+// the scheme from the event.
+type URLFilter struct {
+	// If it's blank/empty, it will handle any scheme.
+	Scheme string
+}
```

Comment:
I suggest

// URLFilter filters for [URLEvent].
type URLFilter struct {
	// Scheme must equal the URLEvent scheme to match. The empty Scheme matches all URLEvents.
	Scheme string
}